xtask/src/main.rs

```diff
@@ -0,0 +1,45 @@
+use clap::{Parser, Subcommand};
+
+#[derive(Parser)]
+#[command(name = "xtask", version, about = "fastn xtask CLI")]
+struct Cli {
+    #[command(subcommand)]
+    command: Commands,
+}
+
+#[derive(Subcommand)]
+enum Commands {
+    NewApp { name: String },
+    BuildWasm,
```

Comment:
Command help texts etc are missing. Run cargo xtask --help and put it in the PR description.

Reply:
We should not use `clap` for xtask. This increases compile time. https://github.com/tursodatabase/libsql/tree/main/xtask is a good inspiration.